README.md

```diff
@@ -71,12 +71,12 @@ DeviceHive license
 ------------------
 
 [DeviceHive] is developed by [DataArt] Apps and distributed under Open Source
-[MIT license](http://en.wikipedia.org/wiki/MIT_License). This basically means
+[Apache 2.0](https://en.wikipedia.org/wiki/Apache_License). This basically means
 you can do whatever you want with the software as long as the copyright notice
 is included. This also means you don't have to contribute the end product or
 modified sources back to Open Source, but if you feel like sharing, you are
 highly encouraged to do so!
 
-Copyright (C) 2015 DataArt Apps
+&copy; Copyright 2013 DataArt Apps &copy; All Rights Reserved
```

Comment:
@BoldinovaNatalya @iilunin @tmatvienko @aleksey-suprun i think we need to keep 2015, and should not change to 2013.